index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Harmattan&family=Inter:wght@100&family=Poppins:wght@200&family=Roboto+Serif:wght@100&family=Roboto:ital,wght@0,400;1,100&display=swap"
+      rel="stylesheet"
+    />
+
+    <link rel="stylesheet" href="CSS/style.css" />
+    <title>My-Portfolio</title>
+  </head>
+  <body>
+    <section>
+      <header>
+        <nav>
+          <div class="mobile-nav">
+            <a href="#" class="name_logo">Gidiz Logo</a>
```

Comment:
[REQUIRED ] Please stick with [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) here you can see that the distance on  top of the logo  is larger than the template

![image](https://user-images.githubusercontent.com/74744787/158690700-ab9cf7a6-df67-4675-994f-cc7b9a336469.png)

![image](https://user-images.githubusercontent.com/74744787/158690730-c6822bec-9bda-4f2c-a600-1c493ab1fb52.png)